src/module/components/ngx-auth-firebaseui-user/user.component.html

```diff
@@ -50,9 +50,7 @@
                    placeholder="Name"
                    [formControl]="updateNameFormControl">
             <mat-icon matSuffix>person</mat-icon>
-            <mat-hint align="end" aria-live="polite">
-              {{updateNameFormControl.value?.length}} / 25
-            </mat-hint>
+            <mat-hint align="end" aria-live="polite"> {{ updateNameFormControl.value?.length }} / {{ config.nameMaxLength }} </mat-hint>
```

Comment:
don't you think that we need some default values... per example for `config.nameMaxLength` .. ?

Reply:
There is default ones in `defaultAuthFirebaseUIConfig`

Reply:
you have right! But the default value were not working because min and max have been initialized on changes hook.

I've just changed the hook to on init

Reply:
You need both in fact. ngOnInit for the case where min/max is not provided and ngOnChanges for the case they are provided and might change over time.